src/components/network/network-panel.js

```diff
@@ -183,3 +209,32 @@ genotet.NetworkPanel.prototype.tooltipEdge = function(edge) {
   this.setEdgeInfo_(edge, tooltip);
   tooltip.find('.close').remove();
 };
+
+/**
+ * Hides the info of the node if one of the nodes in the network.
+ * @param {!Array<string>} genes Genes to hide.
+ */
+genotet.NetworkPanel.prototype.hideNodeInfo = function(genes) {
+  var info = this.container.find('#node-info');
+  var name = info.children('#name').children('span').text();
+  genes.forEach(function(gene) {
+    if (gene == name) {
+      this.hideNodeInfo_();
```

Comment:
When removing several genes with the input blank, this function is to find out whether the gene shown in the panel is removed.